docs/components/base/_index.md

```diff
@@ -91,9 +91,9 @@ Move the base in a straight line across the given distance (mm) at the given vel
 **Parameters:**
 
 - `distance` [(int)](https://docs.python.org/3/library/functions.html#int): The distance to move in millimeters.
-Negative implies backwards.
+Positive implies forwards.
```

Comment:
Why don't we have both for additional clarity?

Reply:
Okay, can do. Though at a quick glance, with it as-is, they'll read "positive...forwards" instead of seeing all the words "positive, forwards, negative, backwards" and needing to actually read the words carefully, so idk if it helps?

Reply:
(done though)